packages/main/src/MenuItem.ts

```diff
@@ -96,6 +97,32 @@ class MenuItem extends UI5Element {
 	@property({ type: Boolean })
 	disabled!: boolean;
 
+	/**
+	 * Defines if a loading indicator would be displayed inside the corresponding sub-menu popover.
+	 *
+	 * Note: If set to <code>true</code> a <code>ui5-busy-indicator</code> component will be displayed into the related to the current <code>ui5-menu-item</code> sub-menu popover.
+	 *
+	 * @name sap.ui.webc.main.MenuItem.prototype.busy
+	 * @type {boolean}
+	 * @defaultvalue false
+	 * @public
+	 * @since 1.13.0
+	 */
+	@property({ type: Boolean })
+	busy!: boolean;
+
+	/**
+	 * Defines the delay in milliseconds, after which the busy indicator will displayed inside the corresponding ui5-menu popover.
```

Comment:
Defines the delay in milliseconds, after which the busy indicator will be displayed inside the corresponding ui5-menu popover.

Reply:
done